### ISSUE TITLE
[Keyboard Navigation– LCS – Index Analysis]: User scans/seeks/update buttons are not accessible while navigating through keyboard. 

### ISSUE THREAD

**Issue by mohit406** [highcharts/highcharts#15542]
### User Experience: 

User will find difficulty to interact with the control if that control is not accessible through keyboard as this will impact the functionality of relevant control and user won't be able to perform the desired functionality over such control. 

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Open #Url: https://lcs.dynamics.com
2. Login with valid credentials.
3. Select "Accessibility Review - D365FOE (Microsoft Corporation)" Project.
4. Project dashboard screen starts appearing.
5. Navigate to "Environment" section and select "Full details" button.
6. New screen starts appearing. Navigate to "Monitoring" section of Main page and select "Environment Monitoring".
7. New screen starts appearing where "Overview" tab is selected. Select "SQL Insights" tab.
8. New screen starts appearing where "Live View" tab is selected. Select "Index Analysis" tab.
9. navigate to controls like user scans/seeks/update.
10. Verify the issue.

### Actual Result:

User scans/seeks/update buttons are not accessible while navigating through keyboard.

### Expected Result:

User scans/seeks/update buttons should be accessible while navigating through keyboard.

**Note:**
This issue also exists with Firefox.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b8492c4eb-c179-40ae-8777-cd044ed725a2%7d

### Reference Links

- Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. 
![MAS2 1 1_Accessible](https://user-images.githubusercontent.com/82201935/115380704-77427880-a1f0-11eb-9ba9-568ea63ca7ef.jpg)
For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
- Please reach out to C&AI Teams channel for any process related queries

**Comment by mohit406** [comment]
A11yMAS,A11ysev2,Accessibility,Benchmark,Benchmark-HCL-LCS-Feb2021,HCL,K4W,MAS2.1.1,Tabstops,Win10-Chrome

**Comment by oysteinmoseng** [comment]
Hi @mohit406, we do not have credentials to access this page. If you are creating these buttons with Highcharts, please look into using [custom accessibility components](https://api.highcharts.com/highcharts/accessibility.customComponents) to add them to the chart keyboard navigation. Otherwise you can place them outside of the chart using standard HTML.

Closing this for now, but feel free to comment if we should reopen.
